src/installs/visualStudio.ts

```diff
@@ -170,6 +172,12 @@ export function getHostTargetArchString(hostArch: string, targetArch?: string, a
         targetArch = hostArch;
     }
 
+    const parsedTargetArch = targetArch.split(",");
+    const version = parsedTargetArch.filter(a => a.includes("=")).map(a => a.split("=")[1]).join(" ").trimEnd();
```

Comment:
my only concern here is because we don't check that they actually say "version=...", but just for = being present, this could lead to a state where their cmakepresets works in vscode, but fails on command line because of a typo for example (so inconsistency in behavior).

Reply:
Updated.